### ISSUE TITLE
Question input fields missing `aria-describedby` attribute referencing the question description

### ISSUE THREAD

**Issue by bion** [civiform/civiform#1879]

**Comment by syntapy** [comment]
I'd like to work on this one

**Comment by bion** [comment]
@syntapy when we chatted on slack it sounded like you're working on other stuff, I'm going to pick this one up since it's related to work I'm already doing

**Comment by jeanniefu** [comment]
Is this the expected behavior? Let's say we have a single question with multiple input fields, e.g. for addresses:

**Enter your address. (question text)**
_Put your primary residence (question help text)_
address: [ ] city: [ ]  state: [ ]  zip: [ ] 

My understanding is that we want to add the `aria-describedby` attribute to each input (e.g. address, city, state, zip), and that that attribute should point to the "Put your primary residence (question help text)" id.

Also when there are validation errors, my understanding is that we want to tag both the question help text ID and the validation error ID (related: https://github.com/seattle-uat/civiform/issues/1878).

**Comment by clouser2** [comment]
That sounds consistent with my understanding. The `aria-describedBy` attribute can take one or multiple elements from what I can tell ([docs](https://www.w3.org/TR/wai-aria/#aria-describedby)).

**Comment by jeanniefu** [comment]
I also chatted with an a11y analyst, who said using aria-describedby sounds good. They also recommended I look into fieldset and legend ([link](https://www.w3.org/WAI/tutorials/forms/grouping/)), but my impression is that the styling of fieldset is quite restrictive ([link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset#styling_with_css)). I will go with the approach I outlined earlier: https://github.com/seattle-uat/civiform/issues/1879#issuecomment-1132372657.

**Comment by jeanniefu** [comment]
Actually I will see if fieldset + legend works for these cases. It sounds like we should be able to override the border and positioning, and restyle it however we want: [link](https://stackoverflow.com/questions/29224061/correct-way-to-group-checkboxes-and-radio-buttons).
